BackofficeBundle/Resources/public/coffee/widget/AbstractConcurrancyCounter.coffee

```diff
@@ -0,0 +1,17 @@
+###*
+ * @class AbstractConcurrancyCounter
+###
+class AbstractConcurrancyCounter extends OrchestraView
+
+  addConcurrency: ->
+    if typeof @currentRankView == 'undefined'
+      @oldRender = @render
+      @currentRankView = 0
+    else
+      @currentRankView = @currentRankView + 1
+    @render = () ->
+      prototype = Object.getPrototypeOf(@)
+      if @currentRankView == prototype.currentRankView
+        @oldRender.apply this, arguments
```

Comment:
I'm a little perturbed, if for a any reason `prototype.currentRankView` has a same value with `@currentRankView` which is defined, so `@oldRender` will not be initialised.